include/kodlab_mjbots_sdk/mjbots_control_loop.h

```diff
@@ -251,6 +249,7 @@ void MjbotsControlLoop<log_type, input_type, robot_type>::Run() {
   robot_->Init();
   lcm_sub_->Init();
   Init();
+  SoftStart::InitializeTimer();
```

Comment:
Is the white space in line 251 messed up or is that a visual clutch on the phone?

Reply:
looks fine to me.